src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -85,7 +86,7 @@ public void equals() {
         assertFalse(ALICE.equals(editedAlice));
 
         // different address -> returns false
```

Comment:
The comment describing the test still refers to address field instead of module code field.